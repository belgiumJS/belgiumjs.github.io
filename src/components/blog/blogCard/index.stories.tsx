import BlogCard from '.';
import type { Meta as MetaObj, StoryObj } from '@storybook/react';

type Story = StoryObj<typeof BlogCard>;
type Meta = MetaObj<typeof BlogCard>;

export const Default: Story = {
	args: {
		slug: 'test',
		lang: 'en',
		title: 'Test',
		description: 'Test',
	},
};

export const WithThumbnail: Story = {
	args: {
		slug: 'test',
		lang: 'en',
		title: 'Test',
		description: 'Test',
		thumbnail: '/static/images/test.jpg',
	},
};

export default { component: BlogCard } as Meta;
