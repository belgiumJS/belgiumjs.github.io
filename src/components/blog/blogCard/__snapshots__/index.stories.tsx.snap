// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blog/blogCard Default smoke-test 1`] = `
<div class="open-sans-normal">
  <div class="blogCard_blogCard__7S3kX">
    <div class="blogCard_content__g11VK">
      <h2>
        Test
      </h2>
      <p>
        Test
      </p>
    </div>
    <a class="blogCard_readMore__jwiad"
       href="/en/blog/test"
    >
      Read more
    </a>
  </div>
</div>
`;

exports[`blog/blogCard WithThumbnail smoke-test 1`] = `
<div class="open-sans-normal">
  <div class="blogCard_blogCard__7S3kX">
    <img src="/static/images/test.jpg"
         alt="Test"
    >
    <a class="blogCard_readMore__jwiad"
       href="/en/blog/test"
    >
      Read more
    </a>
  </div>
</div>
`;
